vm/src/hint_processor/builtin_hint_processor/secp/field_utils.rs

```diff
@@ -90,6 +90,25 @@ pub fn reduce(
     Ok(())
 }
 
+/*
+Implements hint:
+%{
+    from starkware.cairo.common.cairo_secp.secp_utils import pack
+    value = pack(ids.x, PRIME) % SECP_P
+%}
+*/
+pub fn reduce_v2(
```

Comment:
Unit test?

Reply:
As you wish fede [f3c7946](https://github.com/lambdaclass/cairo-vm/pull/1420/commits/f3c79460e2c2c85f442332ceda143c5bb3e0162d)

But, I think unit tests are not so useful in hints